homeassistant/components/recorder/table_managers/statistics_meta.py

```diff
@@ -109,10 +104,17 @@ def _get_from_database(
                 ),
                 orm_rows=False,
             ):
-                statistics_meta = cast(StatisticsMeta, row)
-                id_meta = _statistics_meta_to_id_statistics_metadata(statistics_meta)
-
-                statistic_id = cast(str, statistics_meta.statistic_id)
+                statistic_id = row[INDEX_STATISTIC_ID]
+                row_id = row[INDEX_ID]
+                meta = {
+                    "has_mean": row[INDEX_HAS_MEAN],
```

Comment:
sqlalchemy __getitem__ implementation is an order of magnitude fast because its a tuple under the hood and every name lookup has to be converted